.github/workflows/windows-ci.yml

```diff
@@ -65,3 +65,51 @@ jobs:
 
       - name: Test Slow
         run: poetry run pytest tests ${{ env.pytest-args }} -n auto --runslow -m slow
+
+  rust:
+    runs-on: windows-2019
+    steps:
+      - name: Install sqlite (Windows)
```

Comment:
Should be replaced by a feature enable to bundled sqlite: https://github.com/rusqlite/rusqlite/blob/12fc73d1227472ebb21bbf568a4363509ca34490/Cargo.toml#L48

Reply:
Done in #2720

Reply:
done